### ISSUE TITLE
Layout is called with wrong height value on Mac OS after disabling fullscreen with green button

### ISSUE THREAD

**Issue by hajimehoshi** [hajimehoshi/ebiten#1884]
### Discussed in https://github.com/hajimehoshi/ebiten/discussions/1883

<div type='discussions-op-text'>

<sup>Originally posted by **IlyaFloppy** November 23, 2021</sup>
I am running a game in fullscreen using `ebiten.SetFullscreen(true)`. Layout method is called with width=1919 and height=1200. Then I use green window button to exit fullscreen and `Layout` method keeps getting called with the same width and height. The game looks squeezed vertically. After I resize the window dragging the corner it looks normal and width and height change to 1920 and 1073 respectively.

```go
func (g *Game) Layout(outsideWidth, outsideHeight int) (int, int) {
	fmt.Println("LAYOUT", outsideWidth, outsideHeight)
	return outsideWidth, outsideHeight
}
```

```
// App starts
LAYOUT 1920 1073
// SetFullscreen is called
[1535] LAYOUT 1919 1200
// Green button is used to disable fullscreen
[2643] LAYOUT 1919 1200
// Start changing window size
LAYOUT 1907 1073
LAYOUT 1906 1073
LAYOUT 1905 1073
LAYOUT 1907 1073
LAYOUT 1908 1073
LAYOUT 1909 1073
[14105] LAYOUT 1920 1073
```</div>

**Comment by hajimehoshi** [comment]
https://user-images.githubusercontent.com/16950/142989645-4474a939-4ddc-445b-a8e2-b3f39ff81b27.mov

Video from @IlyaFloppy

**Comment by hajimehoshi** [comment]
CC @IlyaFloppy

Confirmed. I reproduced this issue with these steps:

1. Make the window resizable by `SetWindowResizable`
2. Make the window fullscreen by `SetWindowFullscreen`
3. Click the green button

**Comment by hajimehoshi** [comment]
@IlyaFloppy

OK this should be fixed at `22e64c7df030d882951e4a33dd1e0c0b2b5dd99b`. Could you test this?

**Comment by IlyaFloppy** [comment]
I can confirm this issue has been fixed. Thank you!
